lib/pool.js

```diff
@@ -73,3 +82,4 @@ class Pool {
 }
 
 module.exports = Pool
+module.exports[Symbol.for('pool-symbols')] = { kClients, kQueue }
```

Comment:
If pool also emits `tlsSession` do you need to export the symbol like this?

Reply:
Using an event will not work for how the test is designed, because the event will be fired fewer times, and I'm not able to access the client that has been used for a specific request.

Reply:
I'm not sure I understand either. Why?

```js
function onTLSSession (session) {
  this[kTLSSession] = session
}
```

Reply:
I want to test that the correct session has been propagated to all clients.